apps/web/hooks/useS3.tsx

```diff
@@ -67,11 +67,22 @@ export const S3Provider: React.FC<PropsWithChildren<Props>> = ({
 		isMounted.current = true;
 		if (data.keys.bucketUrl) return;
 
-		if ((Provider[data.type] as Provider) === Provider.s3) {
-			data.keys.bucketUrl = `https://${data.keys.Bucket}.s3.${data.keys.region}.amazonaws.com`;
-		} else if ((Provider[data.type] as Provider) === Provider.backblaze) {
-			data.keys.bucketUrl = `https://${data.keys.Bucket}.s3.${data.keys.region}.backblazeb2.com`;
-		}
+		switch (Provider[data.type] as Provider) {
+			case Provider.s3:
+			  data.keys.bucketUrl = `https://${data.keys.Bucket}.s3.${data.keys.region}.amazonaws.com`;
+			  break;
+			case Provider.backblaze:
+			  data.keys.bucketUrl = `https://${data.keys.Bucket}.s3.${data.keys.region}.backblazeb2.com`;
+			  break;
+			case Provider.wasabi:
+			  data.keys.bucketUrl = `https://${data.keys.Bucket}.s3.${data.keys.region}.wasabisys.com`;
+			  break;
+			case Provider.digitalocean:
+			  data.keys.bucketUrl = `https://${data.keys.Bucket}.${data.keys.region}.digitaloceanspaces.com`;
+			  break;
+			default:
+			  break;
```

Comment:
this is not needed, since if you look into the bucket creation process (`pages/new/digitalocean`) you'll see that we already provide a `bucketUrl` to be stored in our database. So yeah we can remove this :)